example/src/redux/store.js

```diff
@@ -0,0 +1,26 @@
+import { createStore, applyMiddleware, compose, combineReducers as CR } from 'redux';
+import { fetchMiddleware, configureMergeState, wrapCombineReducers } from 'redux-recompose';
+import thunk from 'redux-thunk';
+
+import hearthstone from './hearthstone/reducer';
+
+// Breaking in r-r v2.0: need to specify how do we merge (removed coupled seamless-immutable)
+configureMergeState((state, diff) => state.merge(diff));
```

Comment:
I don't understand very well what is this.

Reply:
I believe this defines how the redux state will be changed by the upcoming data, so you can use seamless-immutable's merge or just merge it old school style like:
```
configureMergeState((state, diff) => ({ ...state, ...diff });
```

Reply:
Since effects by default used `state.merge`, these ones introduce coupling with `seamless-immutable`. The idea here is to specify how these effects merges state with diffs.